### PR TITLE
Fix/realign pv values. TM-909

### PR DESCRIPTION
**Pull request by rtirserio**
Maps data to new FSBid data structure. Requires a new version of the mock (docker-compose build after bringing it down).

### CODE REVIEW

File: talentmap_api/fsbid/services.py [hunk 1]
```diff
@@ -190,24 +197,24 @@ def fsbid_pv_to_talentmap_pv(pv):
...
         "languages": [
             {
-                "language": pv["language1"],
-                "reading_proficiency": pv["reading_proficiency_1"],
-                "spoken_proficiency": pv["spoken_proficiency_1"],
-                "representation": pv["language_representation_1"]
+                "language": pv["lang1"],
+                "reading_proficiency": "",
+                "spoken_proficiency": "",
+                "representation": ""
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services.py]: Do we not get these three data points?

File: talentmap_api/fsbid/services.py [hunk 2]
```diff
@@ -190,24 +197,24 @@ def fsbid_pv_to_talentmap_pv(pv):
...
-        "bureau": pv["bureau"],
-        "organization": pv["organization"],
-        "tour_of_duty": pv["tour_of_duty"],
+        "id": pv["fv_seq_number"],
+        "grade": pv["pos_crade_code"],
+        "skill": pv["pos_skill_desc"],
+        "bureau": pv["bureau_desc"],
+        "organization": pv["post_org_country_state"],
+        "tour_of_duty": pv["tod"],
         "languages": [
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services.py]: Does this model account for multiple languages? I see in the mock-fsbid data that there's both `lang1` and `lang2` fields.

File: talentmap_api/fsbid/services.py [hunk 3]
```diff
@@ -218,26 +225,26 @@ def fsbid_pv_to_talentmap_pv(pv):
         },
         "current_assignment": {
             "user": pv["incumbent"],
-            "estimated_end_date": datetime.strptime(pv["ted"], "%m/%Y")
+            "estimated_end_date": pv["ted"]
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services.py]: Should we assume -5 (`...T05:00:00Z`) timezone to be consistent with our other dates?

Screen Shot 2019-06-24 at 11 26 52 AM

File: talentmap_api/fsbid/services.py (hunk 1 above)
Comment (rtirserio) [reply talentmap_api/fsbid/services.py]: correct, those fields do not come over in the FSBid response

File: talentmap_api/fsbid/services.py (hunk 2 above)
Comment (rtirserio) [reply talentmap_api/fsbid/services.py]: it does, ill add another block for the second language

File: talentmap_api/fsbid/services.py (hunk 3 above)
Comment (rtirserio) [reply talentmap_api/fsbid/services.py]: updated

**Pull request merged**